docs/releases/temporary-versions/version-v12.0.0b1.md

```diff
@@ -0,0 +1,184 @@
+# InvenioRDM v12.0.0b1
+
+_2023-04-13_
+
+_Beta release_
+
+We're happy to announce the release of InvenioRDM v12.0.0b1. The release is a beta release which is an interim release until v12.0.
+
+## Try it
+
+- [Demo site](https://inveniordm.web.cern.ch)
+
+- [Installation instructions](https://inveniordm.docs.cern.ch/install/)
+
+## What's new?
+
+### Usage statistics
+
+A major new feature in this release is the integration of `invenio-stats`, a powerful and flexible extension for measuring usage statistics of InvenioRDM records. With `invenio-stats`, you can gain valuable insights into how your InvenioRDM records are being used, which can help you make informed decisions about how to improve your application and better serve your users. Usage statistics that are compliant with the [MakeDataCount](https://makedatacount.org/) and [COUNTER](https://www.projectcounter.org/) standards. This allows for better tracking and reporting of usage metrics for research data, helping to ensure transparency and accountability in research data management.
+
+Some of the key advantages of using invenio-stats for measuring usage statistics include:
+
+- Detailed information: With invenio-stats, you can track a wide variety of usage statistics, including downloads, views, and citations. You can also view statistics broken down by date range, and other parameters, giving you detailed insights into how your records are being used.
+
+- Customizable metrics: In addition to the default metrics provided by invenio-stats, you can also define your own custom metrics to track specific usage patterns or behaviors.
+
+Usage statistics are displayed in the record landing page and records' search:
+
+![](./v12.0.0/stats-landing-page.png)
+
+### Community submission and required review
+
+With InvenioRDM v12.0.0b1, you can now change if a review is always required when submitting a new record to your community, or if curators, managers and owners can submit a record without the review process. In this case, a review will be created anyway, but it will be automatically accepted. Community manager or owner have the ability to enable or disable this feature for their communities through the community manager interface.
+
+
+![](./v12.0.0/community-curation-policy.png)
+
+When enabled, the community submission without required review option will be available to users when submitting their research outputs to the community. If disabled, this option will not be available, and users will need to go through the traditional peer review process in order to publish their work.
```

Comment:
maybe add here for completeness 2 screenshots with what users will see in the deposit form? i.e show visually what you describe?

Reply:
I am not sure if it is that noticeable. Maybe also would be better to give it a separate doc page ? We started to these "feature" list with something like user tutorial, maybe it would fit there more rather than the release notes?

Reply:
I totally agree, I just anticipate some confusion on what the user should expect from just reading this.... We can enhance the notes later or better, add a new doc page for that :)